invenio_communities/communities/services/service.py

```diff
@@ -714,6 +714,25 @@ def on_relation_update(self, identity, record_type, records_info, notif_time):
 
         return True
 
+    @unit_of_work()
+    def bulk_add_to_parent(self, identity, comm_slugs, new_parent, uow=None):
```

Comment:
```suggestion
    def bulk_add_parent(self, identity, comm_slugs, new_parent, uow=None):
```

Reply:
I disagree on this one, semantically `community_service.bulk_add_parent` seems it's adding parents in bulk. We're adding records, in bulk, to one parent (`community_service.bulk_add_to_parent)`. But I'm open to other suggestions.